src/test/java/greencity/service/impl/BreakTimeServiceImplTest.java

```diff
@@ -0,0 +1,104 @@
+package greencity.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.*;
+
+import greencity.GreenCityApplication;
+import greencity.entity.BreakTime;
+import greencity.exception.BadRequestException;
+import greencity.exception.NotFoundException;
+import greencity.repository.BreakTimeRepo;
+import java.time.LocalTime;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@RunWith(MockitoJUnitRunner.class)
+@SpringBootTest(classes = GreenCityApplication.class)
+public class BreakTimeServiceImplTest {
+    @Mock
+    private BreakTimeRepo breakTimeRepo;
+
+    @InjectMocks
+    private BreakTimeServiceImpl breakTimeService;
+
+    private BreakTime validBreakTime = BreakTime.builder()
+        .id(13L)
+        .startTime(LocalTime.of(8, 0))
+        .endTime(LocalTime.of(19, 0))
+        .build();
+
+    private BreakTime notValidBreakTime = BreakTime.builder()
+        .id(13L)
+        .startTime(LocalTime.of(12, 0))
+        .endTime(LocalTime.of(10, 30))
+        .build();
+
+    @Test
+    public void saveValidBreakTime() {
+        when(breakTimeRepo.save(validBreakTime)).thenReturn(validBreakTime);
+
+        assertEquals(validBreakTime, breakTimeService.save(validBreakTime));
+    }
+
+    @Test(expected = BadRequestException.class)
+    public void saveNotValidBreakTime() {
+        breakTimeService.save(notValidBreakTime);
+    }
+
+    @Test
+    public void findByExistingId() {
+        when(breakTimeRepo.findById(validBreakTime.getId())).thenReturn(Optional.of(validBreakTime));
+
+        assertEquals(validBreakTime, breakTimeService.findById(validBreakTime.getId()));
+    }
+
+    @Test(expected = NotFoundException.class)
+    public void findByNotExistingId() {
+        when(breakTimeRepo.findById(anyLong())).thenReturn(Optional.empty());
+
+        breakTimeService.findById(13L);
+    }
+
+    @Test
+    public void deleteByExistingId() {
+        when(breakTimeRepo.findById(validBreakTime.getId())).thenReturn(Optional.of(validBreakTime));
+
+        assertEquals(validBreakTime.getId(), breakTimeService.deleteById(validBreakTime.getId()));
+
+        verify(breakTimeRepo, times(1)).delete(validBreakTime);
```

Comment:
Consider about removing this blank line between this two operations, these operations can easily be grouped